### PR TITLE
feat(rxPaginate): Persist items per page accross applications.

### LINKED ISSUE #742: rxPaginate should persist the user-selected "items per page"`
Right now when a user selects a value for "items per page", it is global to the app it was selected it, but it is not persisted.

This should become global to *all* apps, and be persisted with LocalStorage. By persisting with LocalStorage, it will automatically become global to all apps.

### PR DESCRIPTION
**Pull request by megalord**
Fixes #742

### CODE REVIEW

File: src/rxPaginate/rxPaginate.js [hunk 1]
```diff
@@ -111,8 +109,10 @@ angular.module('encore.ui.rxPaginate', [])
             itemSizeList.splice(index, 0, itemsPerPage);
         }
 
+        var selectedItemsPerPage = parseInt(LocalStorage.getItem('rxItemsPerPage'));
+
         // If the user has chosen a desired itemsPerPage, make sure we're respecting that
-        if (!_.isUndefined(selectedItemsPerPage) && _.contains(itemSizeList, selectedItemsPerPage)) {
+        if (!_.isNaN(selectedItemsPerPage) && _.contains(itemSizeList, selectedItemsPerPage)) {
```
Comment (parlarjb) [inline src/rxPaginate/rxPaginate.js]: There's actually a pre-existing bug here, maybe you can fix it. If `opts.itemsPerPage` exists (i.e. the dev has requested a specific number to show up on this given page), that value will be ignored.

Would you mind putting a fix for that into this PR?

Comment (megalord) [inline src/rxPaginate/rxPaginate.js]: most certainly

File: src/rxPaginate/README.md [hunk 2]
```diff
@@ -2,6 +2,10 @@
 
 Used for adding pagination around a data object.
 
+# Persistence
+
+The user's preference for the number of items to display per page will be persisted across applications using localstorage. This preference is set whenever the use selects a new number to show.
```
Comment (parlarjb) [inline src/rxPaginate/README.md]: You're missing an "r" here in "user": `whenever the use selects`

**Comment by parlarjb** [comment]
LGTM

**Pull request merged**